src/lib/jsonql/promises.js

```diff
@@ -1,5 +1,11 @@
 import { equalsIgnoreCase, slugify } from "@/promisetracker/utils";
 
+const formatLocation = (latlng) => {
+  if (!latlng) return [process.env.DEFAULT_LAT, process.env.DEFAULT_LONG];
+  const formatted = latlng.split(",");
+  return [parseFloat(formatted[0]), parseFloat(formatted[1])];
+};
```

Comment:
Location is optional so I think work backwards:

1. If latlng, process and ensure valid, if valid, return processed values,
2. If not valid or no latlng altogher, check if we have default lat and long,
3. If we do have default values, return default values, otherwise, return null/undefined => Radar shouldn't be shown for promise.